EMacco/GoBanka
# #168913541 User home

#### What does this PR do?
This PR allows a user to be able to see details of his account as well as transaction history

#### Description of Task to be completed?
- Show account number and balance
- Show past transactions
- Buttons at the bottom for withdrawal, deposit, and logout

#### How should this be manually tested?
- Checkout to this branch
- Login using email and password
- You should see account number, balance, and transaction history

#### What are the relevant pivotal tracker stories?
[#168913541](https://www.pivotaltracker.com/story/show/168913541)

#### Screenshots (if appropriate)
